workflow/rocoto/workflow_tasks.py

```diff
@@ -578,6 +633,11 @@ def _fcst_cycled(self):
         dep = rocoto.add_dependency(dep_dict)
         dependencies = rocoto.create_dependency(dep=dep)
 
+        if self.app_config.do_jediocnvar:
+            dep_dict = {'type': 'task', 'name': f'{self.cdump}ocnanalrun'}
+            dep = rocoto.add_dependency(dep_dict)
+            dependencies = rocoto.create_dependency(dep=dep)
+
```

Comment:
Does `gldas` really need to wait for ocean analysis jobs?

Reply:
rather instead the fcst needs to wait for ocnanal

Reply:
@aerorahul : What @CoryMartin-NOAA said, forecast waiting for ocnanalrun.

Reply:
yes. I understand that dependency (ocnanalrun to complete for fcst to start).  Not sure this is going to be the case if `gldas` is turned off.